### DISCUSSION TITLE
Ubuntu 22.04 LTS VPN not contactable until first graphical login

### DISCUSSION THREAD

**Discussion by bucklevision** [angristan/openvpn-install#1405] [Help]
I have struck an interesting problem when OpenVPN is installed by this script, when I boot the VM containing this installation, I cannot get VPN connections until I log into the VM via the graphical login page. As soon as that is done, there are no problems. I assume this is because the server process is not initialised until that login level, but should be at boot. 

Is this as-designed? Where should I look to change this?

**Comment by angristan** [comment]
That's weird, you can check with `systemctl status openvpn-server@server` to see if it's up before the GUI or not
